androidApp/src/main/java/com/projectnewm/compose/NEWMApp.kt

```diff
@@ -0,0 +1,90 @@
+package com.projectnewm.compose
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.navigation.NavController
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import com.projectnewm.R
+import kotlinx.coroutines.flow.collect
+
+@Composable
+internal fun NEWMApp(navController: NavHostController = rememberNavController()) {
+    Scaffold(
+        bottomBar = {
+            val currentRootScreen by navController.currentRootScreenAsState()
```

Comment:
Is there any way to hoist this state up a level so it gets remembered and we don't have to go find the entry tab each time?

Reply:
We can, the next PR that sets up the nav flow will have it, good call out